frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.test.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { StudioTableLocalPagination } from './StudioTableLocalPagination';
+import { columns, rows } from '../StudioTableRemotePagination/mockData';
+
+describe('StudioTableLocalPagination', () => {
+  const paginationProps = {
+    pageSizeOptions: [5, 10, 50],
+    pageSizeLabel: 'Rows per page',
+    nextButtonText: 'Next',
+    previousButtonText: 'Previous',
+    itemLabel: (num) => `Page ${num}`,
+  };
+
+  it('renders the table with columns and rows', () => {
+    render(<StudioTableLocalPagination columns={columns} rows={rows} />);
+
+    expect(screen.getByRole('columnheader', { name: 'Name' })).toBeInTheDocument();
+    expect(screen.getByRole('columnheader', { name: 'Created by' })).toBeInTheDocument();
+    expect(
+      screen.getByRole('cell', { name: 'Coordinated register notification' }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('cell', { name: 'The Norwegian Directorate of Health' }),
+    ).toBeInTheDocument();
+  });
+
+  it('renders the complete table when isSortable is set to false', () => {
+    render(<StudioTableLocalPagination columns={columns} rows={rows} isSortable={false} />);
+    expect(
+      screen.getByRole('cell', { name: 'Coordinated register notification' }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('cell', { name: 'Application for a certificate of good conduct' }),
+    ).toBeInTheDocument();
+  });
+
+  it('triggers sorting when a sortable column header is clicked', async () => {
+    render(<StudioTableLocalPagination columns={columns} rows={rows} isSortable />);
+
+    await userEvent.click(screen.getByRole('button', { name: 'Name' }));
+
+    const firstBodyRow = screen.getAllByRole('row')[1];
+    expect(
+      within(firstBodyRow).getByRole('cell', { name: 'A-melding – all forms' }),
+    ).toBeInTheDocument();
+
+    const secondBodyRow = screen.getAllByRole('row')[2];
+    expect(
+      within(secondBodyRow).getByRole('cell', { name: 'Application for VAT registration' }),
+    ).toBeInTheDocument();
+  });
```

Comment:
This is an alternative way to query multiple rows. This comment was to inform you about alternatives. 😊

```suggestion
    const [, firstBodyRow, secondBodyRow]= screen.getAllByRole('row');
    expect(
      within(firstBodyRow).getByRole('cell', { name: 'A-melding – all forms' }),
    ).toBeInTheDocument();
   
    expect(
      within(secondBodyRow).getByRole('cell', { name: 'Application for VAT registration' }),
    ).toBeInTheDocument();
  });
```

Reply:
Looks neater - fixed 😊